plugin/src/test/groovy/com/novoda/test/TestAndroidKotlinProject.groovy

```diff
@@ -0,0 +1,78 @@
+package com.novoda.test
```

Comment:
This is basically a copy of `TestAndroidProject`. Changes are marked.

Reply:
It is not too important but with the comments I put below, it is possible that these can be 1 single class with different plugin configuration. 

Then you can have `TestKotlinProject.forAndroid()` to configure the android plugin.

Reply:
I tried that first but failed (see my comment below). Any help is appreciated since I also would like to get rid of this duplication.

Reply:
Just merged it. On a second thought, I don't think it is worth. Because that way, it will be more complicated.